modules/st2-api/api.js

```diff
@@ -192,7 +192,8 @@ export class API {
       params: query,
       headers,
       transformResponse: [ function transformResponse(data, headers) {
-        if (typeof data === 'string' && headers['content-type'] === 'application/json') {
+        if (typeof data === 'string' && headers['content-type'] &&
+            headers['content-type'].indexOf('application/json') >-1) {
```

Comment:
Minor comment, I'd suggest a space before the -1.

Reply:
Done :)